Update OTLP docs to use consistent `applies_to` tags (#5362)
elastic/docs-content: manage-data/data-store/data-streams/tsds-ingest-otlp.md

The endpoint is available under `/_otlp/v1/metrics`.

## Prerequisites

> You need {{stack}} 9.2 or later (the OTLP endpoint is available on ECE, ECK, and self-managed deployments that run this stack version).

Reviewer: is the dependency specific on the elasticsearch version? people use the entire stack of the same version but we can specify if we know

```suggestion
An {{es}} cluster running {{stack}} 9.2 or later. The OTLP endpoint is available on ECE, ECK, and self-managed deployments that run this stack version.
```
